test/runtests.jl

```diff
@@ -31,6 +35,10 @@ using Test
 
     @test [0,0,0] == @_ data |> map(_1.y + _2, __, [-1,-2,-3])
 
+    @test 3 == @_ [[1], [1,2], [1,2,3]] |>
+                                map(_[_[end]], __) |>
+                                __[end]
```

Comment:
Great, thanks!  Could you add an explicit test in the "Underscores lowering" testset as well?

Reply:
Will do.

Reply:
I've now added a few.